Tests/SwiftCliCoreTests/ChartConstructorTest.swift

```diff
@@ -0,0 +1,108 @@
+import Foundation
+import XCTest
+@testable import SwiftCliCore
+
+
+class TopLevelPrinterTests: XCTestCase {
+    
+    // Top Level Tests
+    func testPrintTopLevelMany() throws {
+        let result = ChartConstructor().make_top_row(width: 4)
+        XCTAssertEqual(result, "┌─┬─┬─┬─┐\n")
```

Comment:
This is a great start on writing tests! If you have time, I would recommend looking at using `Nimble` as a test dependency for more expressive assertions.